Update sdk_authentication.md (#2089)
braze-inc/braze-docs: _docs/_developer_guide/platform_wide/sdk_authentication.md

#### What happens if a JWT expires in the middle of a user's session? {#faq-jwt-expiration}

Should a user's token expire mid-session, the SDK has a [callback function][7] it will invoke to let your app know that a new JWT token is needed to continue sending data to Braze.

#### What happens if my server-side integration breaks and I can no longer create a JWT? {#faq-server-downtime}

If your server is not able to provide JWT tokens or you notice some integration issue, you can always disable the feature in the Braze Dashboard.

Once disabled, any pending failed SDK requests will eventually be retried by the SDK and accepted by Braze.

#### Why does this feature use Public/Private keys instead of Shared Secrets? {#faq-shared-secrets}

When using Shared Secrets, anyone with access to that shared secret (i.e. the Braze Dashboard page) would be able to generate tokens and impersonate your end-users.

Instead, we use Public/Private Keys so that not even Braze Employees (let alone your Dashboard users) have access to your Private Keys.

#### How will rejected requests be retried? {#faq-retry-logic}

> When a request is rejected, the SDKs will invoke a callback allowing you do refresh the user's JWT signature. Requests will retry periodically using an exponential backoff approach. After 50 consecutive failed attempts, retries will be paused until the next session start. Each SDK also has a method to manually request a data flush.

Author: ```suggestion
When a request is rejected because of an authentication error, the SDKs will invoke a your callback used to refresh the user's JWT signature. 

Requests will retry periodically using an exponential backoff approach. After 50 consecutive failed attempts, retries will be paused until the next session start. Each SDK also has a method to manually request a data flush.
```
